server/lorax_server/models/custom_modeling/flash_neox_modeling.py

```diff
@@ -27,11 +27,8 @@
 from transformers.models.gpt_neox import GPTNeoXConfig
 from typing import Optional, List, Tuple
 
-# vllm imports
-import vllm_cache_ops
-import vllm_attention_ops
-
 from lorax_server.utils.flash_attn import attention
+from lorax_server.utils import paged_attn
```

Comment:
simple design question: why not using flash_attn prefix as with paged_attn
so it would be easier to understand that flash attention is used
if its only attention there could be missunderstandings which one it is

Reply:
Good point! I'll update so it will be called as `flash_attn.attention` instead.

Reply:
Updated.

Reply:
LGTM